Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -34,13 +34,13 @@ div#attachment-container {
     width: 338px;
 #endif
     gap: 0;
-    background-color: -apple-system-quaternary-fill;
+    background-color: -apple-system-opaque-quaternary-fill;
 #else
     --icon-margin: 14px;
     gap: 4px;
     --information-margin-end: 24px;
     width: 266px;
-    background-color: -apple-system-tertiary-fill;
```

Comment:
These 2 colors were private fwiw:
https://searchfox.org/wubkat/rev/f57f00ab6a22c7b846e9dda0fa296da77aa48e23/Source/WebCore/css/parser/CSSParserIdioms.cpp#42-47

I think they were added for attachment. Can we replace them altogether? Can we make the new ones private too?

Reply:
These colors were chosen by our designers, and were even added to AppKit at about the same time. So I doubt we can replace them.
Or am I missing something, what did you have in mind?

But if we keep them, yes I should make the new derived "opaque" ones private as well.

Reply:
Can we make the new opaque ones private as well?

Btw, can you file a bug about color-mix() not working dynamically with system colors? Ideally we would use it here

Reply:
I've made the new opaque colors private.

About `color-mix()`: I'm not sure anymore that it was incorrect, I'll have another look before filing a bug.
But in any case, after experimenting I prefer using `blendSourceOver` here, to ensure a fully opaque background which `color-mix()` wouldn't give us, and to be consistent with what iOS was already doing.

Reply:
Why does color-mix() not give an opaque color? It should if you use it correctly

Reply:
I did something like `color-mix(in srgb, quaternary, canvas x%)` (with x at 50% or more), which I understood as computing a weighted sum.
Knowing that quaternary.alpha=0.05 and canvas.alpha=1, how could I possibly get alpha==1 in the end when x<100%?
Is there a way to achieve something like `blendSourceOver` with `color-mix()`?